README.md

```diff
@@ -114,8 +114,6 @@ $ cd python && py.test -v                           # optional, run python tests
 ```bash
 $ cd ../../python
 $ python setup.py build_ext --inplace
-# necessary if your dev work continues outside of cudf/cpp/build/python
-$ python setup.py install
```

Comment:
Not sure why you removed these instructions. If you are building from source and you WANT to install rather than setting your PYTHON_PATH env variable, you need this.

Reply:
This PR originally added these README lines, but they're actually in the wrong spot, and are now irrelevant.